composer/src/main/kotlin/com/gojuno/composer/Args.kt

```diff
@@ -61,24 +53,24 @@ data class Args(
                 variableArity = true,
                 description = "Key-value pairs to pass to Instrumentation Runner. Usage example: `--instrumentation-arguments myKey1 myValue1 myKey2 myValue2`.",
                 listConverter = InstrumentationArgumentsConverter::class,
-                order = 6
+                order = 5
         )
         var instrumentationArguments: List<String> = listOf(),
 
         @Parameter(
                 names = arrayOf("--verbose-output"),
                 required = false,
                 arity = 1,
-                description = "Either `true` or `false` to enable/disable verbose output for Composer. `false` by default.",
```

Comment:
I liked "Either x or y" part, otherwise it's not obvious that you can pass booleans

Reply:
Okay, will return it.